doc/guides/backporting-to-release-lines.md

```diff
@@ -8,56 +8,54 @@ pad while preparing a release. The branch name is formatted as follows:
 
 ### Active staging branches
 
-| Release Line | Staging Branch |
-| ------------ | -------------- |
-| `v7.x`       | `v7.x-staging` |
-| `v6.x`       | `v6.x-staging` |
-| `v4.x`       | `v4.x-staging` |
+| Release Line | Staging Branch | LTS Status  |
+| ------------ | -------------- | ----------  |
+| `v8.x`       | `v8.x-staging` | Current     |
+| `v7.x`       | `v7.x-staging` | Maintenance |
+| `v6.x`       | `v6.x-staging` | LTS         |
+| `v4.x`       | `v4.x-staging` | Maintenance |
```

Comment:
I didn't realise this was in the guide, I remember @mylesborins suggested we remove it in the original PR, as it will get out of date every few months.

As this info is basically duplicated in [the LTS schedule](https://github.com/nodejs/LTS/#lts-schedule1), maybe link to that in `## Staging branches` and remove the whole `### Active staging branches` section.